tensorflow/compiler/xla/service/gpu/llvm_gpu_backend/gpu_backend_lib.cc

```diff
@@ -93,7 +93,11 @@ static string GetSmName(std::pair<int, int> compute_capability) {
     }
   }
 
-  if (sm_version != compute_capability_version) {
+  // If the current CC isn't supported by LLVM and it is newer then
+  // the max supported LLVM version, do not warn about it. The end
```

Comment:
Can we please add that "Code compiled for SM75 will run on SM80 too".

Reply:
+1, though I'd s/Code/PTX/